components/uploadFormHook.tsx

```diff
@@ -37,91 +37,86 @@ export const UploadFormHook: FC = () => {
 
   return (
     <>
-      <p className="mb-3 text-2xl font-semibold">アップロードフォーム</p>
+      <h1 className="mb-3 text-2xl font-semibold">アップロードフォーム</h1>
       <form onSubmit={handleSubmit(onSubmit)}>
-        <div>
-          <div className="mb-5">
-            <p>お名前</p>
+        <div className="grid grid-cols-1 gap-6 [&>labe:input]:w-80">
+          <label className="group block">
+            <span className="after:content-['*'] after:ml-0.5 after:text-red-500">
+              お名前
+            </span>
             <input
-              className="w-80"
+              className="block w-80"
               {...register('name')}
               placeholder="オープン太郎"
               required={true}
             />
-            <label />
-          </div>
-
-          <div className="mb-5">
-            <p>所属会社</p>
-            {/* フォームのサイズを調整すべき */}
-            {/* <input {...register('company')} placeholder="株式会社オープンアップグループ" required /> */}
+          </label>
+          <label className="block">
+            <span className="after:content-['*'] after:ml-0.5 after:text-red-500">
+              所属会社
+            </span>
             <input
-              className="w-80"
+              className="block w-80"
               {...register('company')}
               placeholder="オープンアップグループ"
               required={true}
             />
-            <label />
-          </div>
-
-          <div className="mb-5">
-            <p>社員番号</p>
+          </label>
```

Comment:
I think company names list pull-down menu is better in this section.
```suggestion
            <select className="block w-80" {...register('company')} required={true}>
              {/* 会社名のDBを読むようにしたい。 */}
              <option value="" />
              <option value="株式会社オープンアップグループ">
                株式会社オープンアップグループ
              </option>
              <option value="株式会社ビーネックステクノロジーズ">
                株式会社ビーネックステクノロジーズ
              </option>
            </select>
          </label>
```

Reply:
Your comment is right.
However the code which you mentioned isn't related this suggestion.

Please make sure what I changed is correct.

Reply:
Please re-check what I changed.
I've already fixed.

Reply:
I mean you mentioned this section.

```html
<div className="mb-5">
  <p>社員番号</p>
```

However, the employ id is not related to the above issues.

I wonder you wanted to point out like this.

```html
          <div className="mb-5">
            <p>所属会社</p>
            {/* フォームのサイズを調整すべき */}
            {/* <input {...register('company')} placeholder="株式会社オープンアップグループ" required /> */}
            <input
              className="w-80"
          </label>
          <label className="block">
            <span className="after:content-['*'] after:ml-0.5 after:text-red-500">
              所属会社
            </span>
            <select
              className="block w-80"
              defaultValue={''}
              {...register('company')}
              placeholder="オープンアップグループ"
              required={true}
            />
            <label />
          </div>
```

That's why your opinion is correct.

Reply:
I don't know why the code below was mentioned.
```html
<div className="mb-5">
  <p>社員番号</p>
``` 

Now,  your fix merges my opinion. Thanks.